bitmapcontainer.go

```diff
@@ -314,11 +316,11 @@ func (bc *bitmapContainer) iaddReturnMinimized(i uint16) container {
 
 func (bc *bitmapContainer) iadd(i uint16) bool {
 	x := int(i)
```

Comment:
why is this being converted into an int, why not operate directly with the uint16 or make it a uint

Reply:
It is entirely possible that it is an anti-pattern from a performance point of view and something that needs to be fixed.

Reply:
Changing it to `x := uint(i)` does not change the assembly in this case.

Bypassing the variable `x` seems to add an extra assembly instruction and is seemingly not cheaper.